src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -30,6 +30,7 @@ var ReactInstanceMap = require('ReactInstanceMap');
 var ReactPortal = require('ReactPortal');
 var { isValidElement } = require('react');
 var { injectInternals } = require('ReactFiberDevToolsHook');
+var HTMLNodeType = require('HTMLNodeType');
```

Comment:
Maybe we could destructure here so we don't have to do property access everytime we check a `nodeType`?

```js
var {
 ELEMENT_NODE,
 DOCUMENT_TYPE_NODE,
 DOCUMENT_FRAGMENT_NODE
} = require('HTMLNodeType');
```
var